CHANGELOG.md

```diff
@@ -2,11 +2,82 @@
 Notable changes since version 42.0.0, read the complete [History of Changes](https://jdbc.postgresql.org/documentation/changelog.html).
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
-
 ## [Unreleased]
-### Changed
-### Added
-### Fixed
+
+## [42.7.6]
+
+#### Features
+• fix: Enhanced DatabaseMetadata.getIndexInfo() method, added index comment as REMARKS property [PR #3513](https://github.com/pgjdbc/pgjdbc/pull/3513)
+• fix: Added back application name setting [PR #3509](https://github.com/pgjdbc/pgjdbc/pull/3509)
+• fix: Deprecated group startup parameters [PR #3613](https://github.com/pgjdbc/pgjdbc/pull/3613)
+
+### Performance Improvements
+• performance: Use IN row values instead of UNION ALL for improved query performance (later reverted) [PR #3510](https://github.com/pgjdbc/pgjdbc/pull/3510)
```

Comment:
```suggestion
• performance: Improve ResultSetMetadata.fetchFieldMetaData by using IN row values instead of UNION ALL for improved query performance (later reverted) [PR #3510](https://github.com/pgjdbc/pgjdbc/pull/3510)
```